src/Host.cpp

```diff
@@ -1843,3 +1845,28 @@ void Host::setDisplayFontFixedPitch(bool enable)
 {
     mDisplayFont.setFixedPitch(enable);
 }
+
+void Host::loadSecuredPassword()
+{
+    auto *job = new QKeychain::ReadPasswordJob(QStringLiteral("Mudlet profile"));
+    job->setAutoDelete(false);
+    job->setInsecureFallback(false);
+
+    job->setKey(getName());
+
+    connect(job, &QKeychain::ReadPasswordJob::finished, this, [=](QKeychain::Job* job) {
+        if (job->error()) {
+            const auto error = job->errorString();
+            if (error != QStringLiteral("Entry not found") && error != QStringLiteral("No match")) {
+                qDebug() << "dlgConnectionProfiles::Host ERROR: couldn't retrieve secure password for" << getName() << ", error is:" << error;
```

Comment:
Fair enough this is a *lamba* function - but shouldn't the error message reflect that it came about in code located in `Host::loadSecordPassword()` rather than `dlgConnectionProfiles::Host` - whatever that means (perhaps it was originally in `Host:Host()`  i.e. the `Host` c'tor?

Reply:
Fixed it up.